storages/domain_style.go

```diff
@@ -0,0 +1,41 @@
+package storages
+
+import (
+	"net/http"
+	"fmt"
+	"github.com/allegro/akubra/log"
+)
+
+const (
+	HOST              = "X-Akubra-Internal-Host-3yeLjyjQNx"
+	BUCKET            = "X-Akubra-Internal-Bucket-lejK0EpVZy"
+	PATH_STYLE_FORMAT = "/%s%s"
```

Comment:
No underscores in go. Mixed caps in all names, even in consts https://golang.org/doc/effective_go.html#mixed-caps